util/logutil/log.go

```diff
@@ -356,3 +358,27 @@ func WithKeyValue(ctx context.Context, key, value string) context.Context {
 	}
 	return context.WithValue(ctx, ctxLogKey, logger.With(zap.String(key, value)))
 }
+
+// TraceEventKey presents the TraceEventKey in span log.
+const TraceEventKey = "event"
+
+// Event records event in current tracing span.
+func Event(ctx context.Context, event string) {
```

Comment:
do we need to abstract some "EventType"  params,  this maybe a "wait-start"、"wake-up incident" like trace point from different insert, select or DDL. Future we could make reports or generated views on these events based on different types?

Reply:
good idea, how about do that after we design a full `EventType` category in the future?